docs/documentation/cn/sql-reference/sql-functions/bitmap-functions/bitmap_from_string.md

```diff
@@ -0,0 +1,54 @@
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# bitmap_from_string
+
+## description
+### Syntax
+
+`BITMAP BITMAP_FROM_STRING(VARCHAR input)`
+
+将一个字符串转化为一个BITAMP，如果输入的字符串不合法，返回NULL.
```

Comment:
Would better describe what's is valid or invalid string input.

Reply:
done